### ISSUE TITLE
iptables-restore rejects LOG syntax

### ISSUE THREAD

**Issue by fukawi2** [fukawi2/husk#11]
The latest iptables-restore on CentOS 5.9 rejects the LOG syntax generated by husk:

```
/tmp # fire
ERROR: The following line was not accepted by the kernel
-A x_NET_ME -m limit --limit 4/minute --limit-burst 3 -j LOG --log-prefix="[x_NET_ME] "
```

Feeding iptables-restore manually shows the problem is with the LOG target

```
/tmp # husk -4 | iptables-restore 
iptables-restore v1.3.5: Unknown arg `LOG'
Error occurred at line: 26
```

**Comment by fukawi2** [comment]
After experimentation, it appears the syntax requirements have been tightened:
- `minute` needs to be `min`
- The equal sign after `--log-prefix` needs to be removed.

Annoyingly, iptables will still accept the rule, only iptables-restore requires the strict syntax.

**Comment by fukawi2** [comment]
Resolved in commit fbf5490
